dotnet/src/Experimental/Process.Abstractions/Models/KernelProcessMapStateMetadata.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Process.Models;
+
+/// <summary>
+/// Process state used for State Persistence serialization
+/// </summary>
+public sealed record class KernelProcessMapStateMetadata : KernelProcessStepStateMetadata
+{
+    /// <summary>
+    /// Process State of Steps if provided
+    /// </summary>
+    [DataMember]
+    [JsonPropertyName("mapState")]
```

Comment:
Should this be `mapState` or `operationState`?

Reply:
Good eye, yes.  There was some churn around this on merging updates.